data/tasks.js

```diff
@@ -50,20 +49,22 @@ const tasksDataFunctions = {
       constants.stringLimits["textBody"]
     );
     utils.validateDate(dateAddedTo, "dateAddedTo");
-    utils.validateDate(dateDueOn, "dateDueOn");
     utils.validatePriority(priority);
     utils.validateStringInputWithMaxLength(
       tag,
       "tag",
       constants.stringLimits["tag"]
     );
+    checked = utils.validateBooleanInput(checked, "checked");
```

Comment:
what if i dont check the checkbox in the UI do we need this validation then ?
add an if statement to check if the value is coming or no and if its coming then do the validation
if its undefined set it to false

Reply:
done